Example/ViewController.swift

```diff
@@ -30,8 +30,7 @@ class ViewController: UITableViewController {
         let cell = tableView.dequeueReusableCell(withIdentifier: "MiniAppCell", for: indexPath)
 
         let miniAppDetail = self.decodeResponse?[indexPath.row]
-        cell.textLabel?.text = miniAppDetail?.name
-        cell.detailTextLabel?.text = miniAppDetail?.description
```

Comment:
Will the detail text label auto-hide when a value is not assigned?

Reply:
Not auto-hide, there will be no value assigned(empty string) as it is optional

Reply:
Okay. On the Android side, we just modified the visibility to reuse the layout for the future (but we made sure that description text is not visible, to reflect that there have been updates on the BE API). You might want to consider the same.

Reply:
Pushed the changes now.